apps/mochi-web/components/Profile/PaymeLinksTable.tsx

```diff
@@ -0,0 +1,64 @@
+import { ColumnProps, Switch, Table, Typography } from '@mochi-ui/core'
+import { format } from 'date-fns'
+import { useFetchPayRequests } from '~hooks/profile/useFetchPayRequests'
+import { useProfileStore } from '~store'
+import { ModelPayRequest } from '~types/mochi-pay-schema'
+import { Amount, PaymeUrl } from './TableColumns'
+
+const Action: ColumnProps<ModelPayRequest>['cell'] = () => <Switch />
+
+export const PaymeLinksTable = () => {
+  const { me } = useProfileStore()
+  const { data: requests = [], isLoading } = useFetchPayRequests({
+    profile_id: me?.id,
+    entity: 'recipient',
+    type: 'payme',
+  })
+
+  return (
+    <Table
+      border
+      wrapperClassName="rounded-t-none border-t-0"
+      isLoading={isLoading || !me?.id}
+      data={requests}
+      columns={[
```

Comment:
Can you specify the width for the column to resemble the design plus add a min-width for the container to support horizontal scrolling on mobile?